third_party/airflow/README.md

```diff
@@ -1,3 +1,64 @@
 # armada-airflow-operator
 
 An Airflow operator for interfacing with the armada client
+
+## Background
+
+Airflow is an open source project focused on orchestrating Direct Acylic Graphs (DAGs) across different compute platforms.  To interface Airflow with Armada, we had to create a armada operator.  This repository is the implementation of the armada operator.
+
+## Airflow
+
+We followed the [airflow documentation](https://airflow.apache.org/) for setting up a simple test server.  
+
+[setup-local-airflow.sh](./setup-local-airflow.sh) demonstrates how to run airflow locally using Airflow's SequentialExecutor.  This is only used for testing purposes.
+
+I suggest creating a ~/airflow/dags folder and copying the dag files under examples in that location.  This allows you to test the DAG in your airflow test server.
```

Comment:
Please ensure we keep the same person and plurality throughout: First person plural seems most sane to me here, so s/I/We/